docroot/themes/custom/mass_theme/overrides/css/hotfix.css

```diff
@@ -108,4 +108,8 @@ body {
   .ma__org-page .stacked-row-content .ma__stacked-row__section:nth-child(2) {
     border: none;
   }
+
+  .ma__org-page .ma__stacked-row__section ~ .ma__stacked-row__section {
+    border-top: unset;
+  }
```

Comment:
@arthurbaghdas I asked @joeg8612 for the clarification of the ticket and confirmed that we're going to remove the top border from `.ma__org-page .stacked-row-content .ma__stacked-row__section` in screen width less than 921px.  It applies to all org page variations.
So, it seems LL.101-114 can be simply replaced with:
```
.ma__org-page .stacked-row-content .ma__stacked-row__section {
   border: none;
}
```

Reply:
HI @ygannett , added proposed change. please review again.

Reply:
@arthurbaghdas These sections in the sample pages still have the borders:
- /orgs/qag-digital-services
- /orgs/qag-test-elected-org-page
- /orgs/qag-organizationboards

<img width="856" alt="contactus" src="https://user-images.githubusercontent.com/9633303/98129616-53c88300-1e87-11eb-94c0-aa06f045165a.png">
<img width="975" alt="featureditems" src="https://user-images.githubusercontent.com/9633303/98129628-57f4a080-1e87-11eb-8f5c-38aa17e29c40.png">
I see LL. 101-106 is applying the style:

```
  .ma__org-page .stacked-row-content .ma__stacked-row__section:nth-child(2) .ma__contact-list,
  .ma__org-page .stacked-row-content .ma__stacked-row__section:nth-child(2) .ma__featured-item-mosaic {
    border-color: #dcdcdc;
    border-top-width: 1px;
    border-top-style: solid;
  }
```

Since LL. 108-110 

```
.ma__org-page .stacked-row-content .ma__stacked-row__section:nth-child(2) {
    border: none;
  }
```

is covered by LL. 112-114

```
  .ma__org-page .stacked-row-content .ma__stacked-row__section {
    border: none;
  }
```

it can be removed.

Let me know for any questions.

Reply:
Makes sense, thanks @ygannett , can you recheck?